pkg/extensions/sync/utils.go

```diff
@@ -678,7 +679,8 @@ func descriptorEqual(desc1, desc2 ispec.Descriptor) bool {
 
 func isSupportedMediaType(mediaType string) bool {
 	return mediaType == ispec.MediaTypeImageIndex ||
-		mediaType == ispec.MediaTypeImageManifest
+		mediaType == ispec.MediaTypeImageManifest ||
+		mediaType == manifest.DockerV2ListMediaType
```

Comment:
wait, what? docker.io supports OCI on the wire right?

Reply:
yes, but here is the upstream mediatype and in the case of docker.io is not OCI